net.gaijin.WarThunder.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: net.gaijin.WarThunder
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: run.sh
+tags:
+  - proprietary
+finish-args:
+  - --socket=x11
+  - --device=dri
+  # =all for controller support
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: warthunder
+    buildsystem: simple
+    build-commands: 
+      - install -D -t "${FLATPAK_DEST}/bin/" ./run.sh
+      - install -D ./apply_extra.sh "${FLATPAK_DEST}/bin/apply_extra"
+      - install -D -t "${FLATPAK_DEST}/share/applications/" ./net.gaijin.WarThunder.desktop
+      - install -D ./net.gaijin.WarThunder-128.png "/app/share/icons/hicolor/128x128/apps/net.gaijin.WarThunder.png"
+      - install -D -t "${FLATPAK_DEST}/share/metainfo/" ./net.gaijin.WarThunder.metainfo.xml
+    sources:
+      - type: extra-data
+        filename: wt.tar.gz
+        url: https://cdnnow-distr.gaijinent.com/wt_launcher_linux_0.9.4.74.tar.gz
+        sha256: 68d4e26c170091f9ede93730f8d258e72af5f325c256ffa6e44fdb9ad0c7d638
+        # curl -s -L -I "URL" | gawk -v IGNORECASE=1 '/^Content-Length/ { print $2 }'
+        size: 19275822
```

Comment:
This is a template.
```yaml
        url: https://aws-yup-distr-03.gaijinent.com/wt_launcher_linux_0.9.4.74.tar.gz
        sha256: 68d4e26c170091f9ede93730f8d258e72af5f325c256ffa6e44fdb9ad0c7d638
        # curl -s -L -I "URL" | gawk -v IGNORECASE=1 '/^Content-Length/ { print $2 }'
        size: 19275822
        x-checker-data:
          type: rotating-url
          url: 'https://warthunder.com/en/download/launcherLinux'
          pattern: 'https://cdnnow-distr.gaijinent.com/wt_launcher_linux_([0-9.]+).tar.gz'
```

Reply:
For what exactly?

Reply:
It's a template to implement flatpak-external-data-checker for WarThunder. Essentially what that does is check if the launcher has an update, submits an MR and merges an hour later if it wasn't manually merged.

This should in theory prevent the self-updater from complaining as much as possible.

Reply:
@Doomsdayrs please add it to make sure the app is not suddenly broken if that URL disappears.

Reply:
I will have to wait until they update their launcher to see.

Currently I am betting it will not work.

Reply:
Is it allowed to create a script that would run daily on the repo to update a file with the sha256 sum of the latest launcher?

Reply:
I believe it's something flatpak-external-data-checker supports as well.